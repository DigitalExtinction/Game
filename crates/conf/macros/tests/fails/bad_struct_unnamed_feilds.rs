use conf_macros::Config;

#[derive(Config)]
struct TestConfig (
    f32,i32
);

fn main() {}
