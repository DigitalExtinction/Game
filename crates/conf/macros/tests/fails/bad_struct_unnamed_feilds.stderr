error: expected a struct with named fields
 --> tests/fails/bad_struct_unnamed_feilds.rs:4:19
  |
4 |   struct TestConfig (
  |  ___________________^
5 | |     f32,i32
6 | | );
  | |_^
